cves/kernel/CVE-2021-3491.yml

```diff
@@ -322,8 +320,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: Two users signed off on the fixing commit
```

Comment:
I would specify that the users that signed off were:

Thadeu Lima de Souza Cascardo <cascardo@canonical.com>
 Jens Axboe <axboe@kernel.dk>

Reply:
Good catch, thanks